CDCgov/prime-reportstream
# 8288 Send Okta user email with all App Insights events

This PR ...

Sends the Okta user email with all App Insights events.

Test Steps:
1. Log into RS
2. Navigate to different pages within RS, filter dates, click on pagination, and let session expire
3. In App Insights, query the logs and notice that the user_AuthenticatedId field is now populated with the signed in users email.
```
let usg_events = dynamic(["*"]);
union pageViews, customEvents, requests
    | where timestamp > ago(4h)
    | where isempty(operation_SyntheticSource)
    | extend name =replace("\n", "", name)
    | where '*' in (usg_events) or name in (usg_events) 
    | where user_AuthenticatedId != ""
    | project timestamp, name, operation_name, user_AuthenticatedId
```

## Changes
![Screenshot 2023-03-06 at 6 05 39 PM](https://user-images.githubusercontent.com/102491809/223308965-a6d3f30e-0432-43e1-b1e0-e8c0e6dfc248.png)


## Checklist

### Testing

## Linked Issues
- Fixes #8288
